backend/src/main/java/de/bund/digitalservice/ris/norms/application/service/NormService.java

```diff
@@ -745,17 +754,13 @@ private void fixTimeline(final NormExpressionEli removedNormExpressionEli) {
     // 2. The removed orphan was at the end of the timeline  (previous-norm is present)
     // The case that only the next-norm is present is not possible because at least the original expression must be present
     if (previousNorm.isPresent() && nextNorm.isPresent()) {
-      final Norm newPreviousManifestation = createNewVersionOfNormService.createNewManifestation(
-        previousNorm.get()
-      );
-      final FRBRExpression previousFrbrExpression = newPreviousManifestation
+      final Norm previousExpression = previousNorm.get();
```

Comment:
Why this change? We do need a new manifestation of that expression because of changing the next version id